tools/ci/after_install.sh

```diff
@@ -12,3 +12,22 @@ if [ "$TEST_SUITE" = "spec:javascript" ]; then
   yarn list
   echo "travis_fold:end:YARN_LOCK"
 fi
+
+if [ "$TEST_SUITE" = "spec" ]; then
+  OLD=`mktemp`
+  NEW=`mktemp`
+
+  bundle exec debride --rails app/controllers/ | cut -d ":" -f1 > "$NEW"
+
+  git checkout  master
+
+  # remove once it's merged
+  gem install debride
```

Comment:
not needed anymore?

Reply:
When I switch at `master` branch I do not have `debride` in `manageiq-ui-classic.gemspec`. So I can either remove it after it's merged or have this PR not working. I'm ok with either one :)